src/coreclr/vm/i386/cgenx86.cpp

```diff
@@ -436,6 +436,10 @@ void StubDispatchFrame::UpdateRegDisplay_Impl(const PREGDISPLAY pRD, bool update
         // This path is hit when we are throwing null reference exception from
         // code:VSD_ResolveWorker or code:StubDispatchFixupWorker
         pRD->ControlPC = GetAdjustedCallAddress(pRD->ControlPC);
+#ifdef FEATURE_EH_FUNCLETS
+        // We need to set EIP to match to ensude Thread::VirtualUnwindCallFrame
```

Comment:
Ah, part of the comment got lost somewhere...

```suggestion
        // We need to set EIP to match ControlPC to ensure Thread::VirtualUnwindCallFrame
        // doesn't fail assertion on GetControlPC(pRD) == GetIP(pRD->pCurrentContext)
        // precondition.
```